src/cart/SConscript

```diff
@@ -98,6 +98,7 @@ def scons():
     denv.AppendUnique(RPATH=['$PREFIX/lib'])
 
     pp_env = Environment(TOOLS=['default', 'extra'])
+    pp_env.Replace(CC=env['CC'])
```

Comment:
Do you still want this?

Reply:
I didn't test without it, but I think we do. if we use the icc preprocessor but use clang compiler, we are certainly going to run into problems because the atomic built-ins have different prototypes in icc and clang.

Reply:
here's what I am talking about, if you search for atomic_load_consume you will see the error. scroll up a few lines you will see icc is mixed with clang. is that supposed to happen?